### PR TITLE
CONCF-597 | Fix styles after changing markup of rail modules

### PR DESCRIPTION
**Pull request by vforge**
/cc @bognix @Warkot

### CODE REVIEW

**Comment by bognix** [comment]
LGTM

File: extensions/wikia/Chat2/css/ChatEntryPoint.scss [hunk 1]
```diff
@@ -64,6 +64,7 @@ body.skin-oasis .ChatMonobookEntryPoint {
 			font-size: 12px;
 			font-weight: normal;
 			line-height: 14px;
+			margin-top: 5px;
```
Comment (Warkot) [inline extensions/wikia/Chat2/css/ChatEntryPoint.scss]: Why u no margin right and bottom?

**Comment by hakubo** [comment]
such random, so numbers :)

**Comment by hakubo** [comment]
other than that...

LGTM

**Comment by Warkot** [comment]
LGTM :+1:

**Pull request merged**